docs/resources/service_v1.md

```diff
@@ -228,7 +228,7 @@ $ terraform import fastly_service_v1.demo xxxxxxxxxxxxxxxxxxxx
 
 ### Required
 
-- **domain** (Block Set, Min: 1) (see [below for nested schema](#nestedblock--domain))
+- **domain** (Block Set, Min: 1) A set of Domain names to serve as entry points for your Service (see [below for nested schema](#nestedblock--domain))
```

Comment:
Interesting. Why are these showing up in the documentation now? Is this because of an update to the tfplugindocs tool?

Reply:
I have been looking through the notes for the tagged releases and nothing has jumped out.  Looking at the code it does seem to be the only TypeSet with a description.  Do you want me to remove this to keep the generated docs consistent?

Reply:
No that's fine that it's there. I checked `master` and the description was already there so it was just weird that it wasn't in the generated docs, but if it's there now then that's fine 🙂